### PR TITLE
holi-theme colors

### LINKED ISSUE #126: Holi Theme
**Describe your theme in detail**
Holi theme is a clean visual studio code dark theme inspired by the festival of colors, Holi.

**Include a screenshot / image**
Holi theme preview

**Color palette**
```
"background" => "#030314FF",
"border" => "#85A4C0FF",
"stroke" => "#2A4555FF",
"ring" => "#D6E7FFFF",
"currStreakNum" => "#5FABEEFF",
"fire" => "#D6E7FFFF",
"sideNums" => "#5FABEEFF",
"currStreakLabel" => "#D6E7FFFF",
"sideLabels" => "#D6E7FFFF",
"dates" => "#85A4C0FF",
```

Are you going to add the theme?
- [x] Check for yes

Looking forward!

### PR DESCRIPTION
**Pull request by AyushSaini00**
## Description
This PR includes [Holi theme](https://github.com/AyushSaini00/holi-theme) to the project.

Fixes #126

### Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [ ] New feature (added a non-breaking change which adds functionality)
- [ ] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [x] Tested locally with a valid username
- [x] Tested locally with an invalid username
- [x] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Holi Theme

Thank you for developing the project. Looking forward!!! 🙌

### CODE REVIEW

**Review by DenverCoder1 — APPROVED** [review]
:tada: Looks great!

Would you like to also add it to the list in https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/themes/README.md?

**Comment by AyushSaini00** [comment]
Sure 👍

File: docs/themes/README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ You can also try out and customize these themes on the [Demo Site](https://githu
 |         `blux`         | ![image](https://user-images.githubusercontent.com/73984717/116656177-7068f200-a9b6-11eb-91c5-5f0f1997720d.png) |
 |        `earth`         | ![image](https://user-images.githubusercontent.com/20955511/119410749-fcd6ae00-bcf1-11eb-8737-c78566e9e8f3.png) |
 |       `deepBlue`       | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png) |
+|       `holi-theme`       | ![image](https://user-images.githubusercontent.com/62163030/129030429-0a4d48aa-d61f-4d0c-ae82-891d1b2476d7.png) |
```
Comment (DenverCoder1) [inline docs/themes/README.md:71]: Remove spaces to keep column widths consistent

```suggestion
|       `deepBlue`       | ![image](https://user-images.githubusercontent.com/20955511/127893911-23e6c0c6-2d97-4028-a2b9-1fa1063db776.png) |
|      `holi-theme`      | ![image](https://user-images.githubusercontent.com/62163030/129030429-0a4d48aa-d61f-4d0c-ae82-891d1b2476d7.png) |
```

**Review by DenverCoder1 — APPROVED** [review]
Awesome! Looks good. :+1: 

I just have a comment about the spacing

**Pull request merged**